Client/Ecosia/AppDelegate/CompositeAppDelegate.swift

```diff
@@ -0,0 +1,57 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+typealias AppDelegateType = UIResponder & UIApplicationDelegate
+typealias AppDelegateLeaf = AppDelegateType
```

Comment:
I'm happy with that too as long as we at least suggest what Design Pattern we based the work upon. That way it would be event easier in case some individual contributor could come up with an event better idea.